cmd/goProbe/goProbe.go

```diff
@@ -118,19 +119,17 @@ func main() {
 
 	// configure api server
 	var (
-		apiServer  *server.Server
+		apiServer  *gpserver.Server
 		apiOptions = []server.Option{
-			server.WithDBPath(config.DB.Path),
 			// Set the release mode of GIN depending on the log level
 			server.WithDebugMode(
 				logging.LevelFromString(config.Logging.Level) == logging.LevelDebug,
 			),
+			server.WithProfiling(config.API.Profiling),
```

Comment:
I trust this will not crash in case the profiling section is not in the config?

Reply:
It's `false` by default and the crash of non-existent `*API` config has been resolved already.